actionpack/lib/action_dispatch/middleware/cookies.rb

```diff
@@ -144,7 +147,7 @@ def cookies_digest
   # * <tt>:tld_length</tt> - When using <tt>:domain => :all</tt>, this option can be used to explicitly
   #   set the TLD length when using a short (<= 3 character) domain that is being interpreted as part of a TLD.
   #   For example, to share cookies between user1.lvh.me and user2.lvh.me, set <tt>:tld_length</tt> to 1.
-  # * <tt>:expires</tt> - The time at which this cookie expires, as a \Time object.
+  # * <tt>:expires</tt> - The time at which this cookie expires, as a \Time or ActiveSupport::Duration object.
```

Comment:
Will this link to the duration class when the doc is generated?

Reply:
Yes, the generated docs do indeed link to the duration class :smile: 

![getimage 1](https://user-images.githubusercontent.com/10774488/30015672-5c164496-916f-11e7-8db0-f67c4e1f3a16.png)